ironfish-cli/src/commands/accounts/pay.ts

```diff
@@ -195,19 +195,26 @@ ${displayIronAmountWithCurrency(
 
     try {
       const result = await client.sendTransaction({
-        amount: ironToOre(amount).toString(),
         fromAccountName: from,
-        memo: memo,
-        toPublicKey: to,
+        receives: [
+          {
+            publicAddress: to,
+            amount: ironToOre(amount).toString(),
+            memo: memo,
+          },
+        ],
         fee: ironToOre(fee).toString(),
         expirationSequence,
       })
 
       stopProgressBar()
 
       const transaction = result.content
+      const recipients = transaction.receives
+        ?.map((receive) => receive.publicAddress)
+        .join(', ')
       this.log(`
-Sending ${displayIronAmountWithCurrency(amount, true)} to ${transaction.toPublicKey} from ${
+Sending ${displayIronAmountWithCurrency(amount, true)} to ${recipients} from ${
```

Comment:
Just double-checking that this ends up being formatted okay

Reply:
This does end up being formatted okay in my local testing as well as in isolation in a TypeScript REPL.